tests/Application/config/services.yaml

```diff
@@ -99,9 +99,6 @@ services:
     App\Subscription\:
         resource: '../src/Subscription'
 
-    App\Subscription\Factory\SubscriptionFactory:
```

Comment:
We can still use a factory service here but the ` app.factory.subscription` doesn't exist if we do not configure any driver.
So It's more complicated like this, cause we need to tag this factory to be used in the repository locator.